book_source/03_topical_pages/93_installation/03_install_OS/04_Installing-PEcAn-OSX.Rmd

```diff
@@ -1,12 +1,33 @@
 ### Mac OSX {#macosx}
 
-These are specific notes for installing PEcAn on Mac OSX and will be referenced from the main [installing PEcAn](Installing-PEcAn) page. You will at least need to install the build environment and Postgres sections. If you want to access the database/PEcAn using a web browser you will need to install Apache. To access the database using the BETY interface, you will need to have Ruby installed.
+These are specific notes for installing PEcAn on Mac OSX and referenced from the [installing PEcAn](Installing-PEcAn) page. 
 
-This document also contains information on how to install the Rstudio server edition as well as any other packages that can be helpful.
+Build environment and Postgres sections are required to install and use PEcAn.
 
+Optional software includes Apache, Rails, and Rstudio. Apache is required to run the BETYdb UI and the web-based version of PEcAn - both are optional. Rails is requred to use the BETYdb web interface. Rstudio is a commonly used IDE for R.
 
 #### Install build environment
 
+##### Option 1: Homebrew
```

Comment:
The official R-project party line is so strongly anti-Homebrew that we might want to think twice about including it as the #1 option -- anyone who comes to this doc without R installed is also someone likely to be asking for help in the channels that are very anti-Homebrew. 

How much of this document could be replaced by "Install R in your favorite way, then run `options(repos = c(getOption("repos"), PEcAn = "pecanproject.r-universe.dev")); install.packages("PEcAn.all")`"?

Reply:
> How much of this document could be replaced 

good question - but out of scope for this PR!

Reply:
moved it to Option 2. I gather that only applies to installing r, not all of the dependencies